lib/types.ts

```diff
@@ -14,25 +16,37 @@ type MergeWithOptionalFields<T, U> = {
   [K in keyof Omit<T, keyof U>]?: NonNullable<T[K]>;
 };
 export type AccountWithKey<T> = { publicKey: PublicKey; account: T };
-export type ProgramVersion = { label: string; programId: PublicKey; idl: Idl };
+export type ProgramVersion = {
+  label: string;
+  programId: PublicKey;
+  idl: Idl;
+};
 export type AutocratProgram = AutocratV0 | AutocratV0_1;
+export type DaoStateV0_1 = MergeWithOptionalFields<
```

Comment:
I don't know that this is the best way to do it BTW. Just that it works.

Reply:
Works for me 😄 I mean I think it works out pretty well tbh